selfify/sidebar/choose_file.js

```diff
@@ -0,0 +1,63 @@
+/*
+Listens for a file being selected, creates a ObjectURL for the chosen file, injects a
+content script into the active tab then passes the image URL through a message to the
+active tab ID.
+*/
+
+// Listen for a file being selected through the file picker
+let inputElement = document.getElementById("input");
```

Comment:
Sorry, seems I was behind on ES6+ best practices :-( On review, const seems the best for all.